### PR TITLE
Add Damion's Corpus Christi Recommendation file

### LINKED ISSUE #19: Create a file for Corpus Christi Recommendations
- [ ] Create a branch
- [ ] Add a commit
- [ ] Open a pull request
- [ ] Have a discussion
- [ ] Seek approval
- [ ] Merge Pull Request

### PR DESCRIPTION
**Pull request by Dscholz23**
Closes #19

### CODE REVIEW

File: Dscholz23-CorpusChristi.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+University, Texas A&M Corpus Christi
+Beach,  Bob Hall Pier
+Kayaking, Laguna Madre
+Baseball, CC Hook's
+
+### Places to avoid
+Robstown
+
+### Fun Facts
+2nd windiest city in USA
```
Comment (w-harrison) [inline Dscholz23-CorpusChristi.md:19]: What's number 1?

**Comment by w-harrison** [comment]
:whale:

File: Dscholz23-CorpusChristi.md (hunk 1 above)
Comment (Dscholz23) [reply Dscholz23-CorpusChristi.md:19]: Chicago

**Review by w-harrison — COMMENTED** [review]
:+1:

**Review by w-harrison — APPROVED** [review]

**Pull request merged**